packages/redux_persist_flutter/redux_persist_flutter.iml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ignore globally (as said in thread comment) and `git rm --cached packages/redux_persist_flutter/redux_persist_flutter.iml`